### ISSUE TITLE
FP: Suggesting Elvis-Operator for <?bool> ?? true

### ISSUE THREAD

**Issue by tlamy** [kalessil/phpinspectionsea#1412]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Ultimate) 3.0.4 |
| Language level | PHP 7.3                        |

Current behaviour
--
![image](https://user-images.githubusercontent.com/1841537/64950784-8034d080-d87c-11e9-8592-fcb060f5ebbc.png)

```php
class ProgramConfig {
    public function getAutoRestart(): ?bool
    {
        return $this->getProperty('autorestart');
    }
}

/** @var ProgramConfig $programConfig */
$restart = $programConfig->getAutoRestart() ?? true;
```
$restart should be whatever getAutoRestart() returns, with `true` as default (when method result is null).

Expected behaviour
--
No inspection message, code is valid

Environment details
---
PhpStorm 2019.2.2 RC
Build #PS-192.6603.26, built on September 5, 2019
Licensed to AdTexx GmbH / Thomas Lamy
Subscription is active until September 23, 2019
Runtime version: 11.0.3+12-b304.56 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.14.6
GC: ParNew, ConcurrentMarkSweep
Memory: 1716M
Cores: 4
Registry: run.processes.with.pty=TRUE
Non-Bundled Plugins: com.ppolivka.gitlabprojects, mobi.hsz.idea.gitignore, BashSupport, com.kalessil.phpStorm.phpInspectionsUltimate, de.espend.idea.php.annotation, izhangzhihao.rainbow.brackets, org.zalando.intellij.swagger, ru.adelf.idea.dotenv

**Comment by kalessil** [comment]
I'll drop this behaviour completely.

**Comment by kalessil** [comment]
Fixed, will be in the hot-fix release I'll  prepare today.
